src/azul/service/storage_service.py

```diff
@@ -32,10 +35,17 @@ def head(self, object_key: str) -> dict:
     def get(self, object_key: str) -> bytes:
         return self.client.get_object(Bucket=self.bucket_name, Key=object_key)['Body'].read()
 
-    def put(self, object_key: str, data: bytes, content_type: Optional[str] = None, **kwargs) -> str:
+    def put(self,
+            object_key: str,
+            data: bytes,
+            content_type: Optional[str] = None,
+            file_name: Optional[str] = None,
```

Comment:
Let's generalize this

```suggestion
            tagging: Optional[Mapping[str, str]] = None,
```